Makefile

```diff
@@ -87,6 +87,7 @@ SHARED_ARGS   := \
 	-Wall -Wpedantic                                              \
 	-Wno-char-subscripts                                          \
 	-Wno-gnu-anonymous-struct                                     \
+	-Wno-c99-extensions                                           \
```

Comment:
Just out of curiosity, which warning does this suppress?

Reply:
```
./astronomy/orbital_elements_body.hpp:165:10: warning: designated initializers
      are a C99 feature [-Wc99-extensions]
        {.t = time,
         ^~~~~~~~~
./astronomy/orbital_elements_body.hpp:166:10: warning: designated initializers
      are a C99 feature [-Wc99-extensions]
         .a = *osculating_elements.semimajor_axis,
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
./astronomy/orbital_elements_body.hpp:167:10: warning: designated initializers
      are a C99 feature [-Wc99-extensions]
         .h = e * Sin(ϖ),
         ^~~~~~~~~~~~~~~
```
…and so on.

Reply:
Ah, right, these are C++20 whereas the flags say 1z=17.